plugin/proto/proto.go

```diff
@@ -15,64 +15,68 @@
 // and notation external plugin.
 package proto
 
+import "github.com/notaryproject/notation-plugin-framework-go/plugin"
+
 // Prefix is the prefix required on all plugin binary names.
 const Prefix = "notation-"
```

Comment:
Will we replace it with `notation-plugin-framework-go/plugin`

Reply:
Done